### DISCUSSION TITLE
Request: Remove the original torrent and content folder after uploading

### DISCUSSION THREAD

**Discussion by mat926** [RogueOneEcho/caesura#76] [Ideas]
I am using the options "copy_transcode_to_content_dir" and "copy_torrent_to". The original files that get created in the output folder can add up and clutter over time. Having the option to remove these files would be nice.

**Comment by RogueOneEcho** [comment]
This isn't something I'd really considered, my initial reaction is this isn't necessary as there are existing ways to work around this:

1. Load torrents into your client from the output directory

Personally, I don't use the `copy_transcode_to_content_dir` option. I simply set the output directory as the `download directory` when loading into Deluge or qBittorrent. I find this helps keep a clear separation between stuff I've download and my own uploads/transcodes.

2. Use hard links

If you enable  the `hard_link` option along with `copy_transcode_to_content_dir` then the transcodes are hard linked from one directory to the other instead of copied. Therefore no additional space is taken up.

---

Those solutions solve the additional space issue but I do see your point about clutter. 

- [ ] Once uploaded and the content copied elsewhere there's no reason to keep the `.torrent` or the transcode in the output directory.

This isn't high on my personal priorities but if others are interested in looking at this I'd certainly consider it.

For now, I'll move this to the [ideas discussion](https://github.com/RogueOneEcho/caesura/discussions/categories/ideas) where I'm tracking interest and feedback on new features. If there's enough interest I may find time for it.

**Reply by RogueOneEcho** [reply]
See also: 

Source: https://github.com/RogueOneEcho/caesura/discussions/77#discussioncomment-11712237

> Also the current workflow [(discussed here)](https://github.com/RogueOneEcho/caesura/issues/85) leaves a lot of trash around that's kind of a pain to clean up. If i'm copying the torrents to another location for auto-add why is it leaving the original behind? why is the .torrent used to check for transcodes not deleted?
